src/Users/README.md

```diff
@@ -61,16 +61,63 @@ if (!$result) {
 }
 ```
 
+### UPDATE OWN BASIC INFO
+```php
+$updatedUser = \ATDev\RocketChat\Users\User::updateOwnBasicInfo([
```

Comment:
@ASergey Do we really need static call here? I think we can create the object via ::me, set all the required data and do the update. I think we should be static calls when there is no object to work with (link listings), which here we have currently logged in user object to update.